docs/software/commands.md

```diff
@@ -638,3 +638,58 @@ enabled by specifying a port via the HTTP_QUERY_PORT config setting.
   value of the `TTL` entry to the current ledger sequence number.
 
   `ledgerSeq` gives the ledger number on which the query was performed.
+
+* **`getledgerentry`**<br>
+  A POST request with the following body:<br>
+
+  ```
+  ledgerSeq=NUM&k=Base64&k=Base64...
+  ```
+- `ledgerSeq`: An optional parameter, specifying the ledger snapshot to base the query on.
+  If the specified ledger is not available, a 404 error will be returned with "Ledger not found\n" message.
+  If this parameter is not set, the current ledger is used.
+- `key`: A series of Base64 encoded XDR strings specifying the `LedgerKey` to query. Keys must be
+  unique and must not be TTL entries, as TTL data is automatically returned when querying a Soroban key.
+
+A JSON payload is returned as follows:
+
+```js
+{
+"entries": [
+     {"entry": "Base64-LedgerEntry", "state": "live", /*optional*/ "ttl": uint32},
+     {"entry": "Base64-LedgerKey", "state": "not-found"},
+     {"entry": "Base64-LedgerEntry", "state": "archived"}
+],
+"ledgerSeq": uint32
+}
+```
+
+- `entries`: A list of entries for each queried LedgerKey. Every key queried is guaranteed to
+  have a corresponding entry returned, and they appear in the same order as they were specified in the request.
+- `entry`: Either the `LedgerEntry` or `LedgerKey` for a given key encoded as a Base64 string. If a key
+  is live or archived, `entry` contains the corresponding `LedgerEntry`. If a key does not exist
+  (including expired temporary entries) `entry` contains the corresponding `LedgerKey`.
+- `state`: One of the following values:
+  - `live`: Entry is live.
+  - `not-found`: Entry does not exist. Either the entry has never existed or is an expired temp entry.
+  - `archived`: Entry is archived, counts towards disk resources.
+- `ttl`: An optional value, only returned for live Soroban entries. Contains
+  a uint32 value for the entry's `liveUntilLedgerSeq`.
```

Comment:
For the sake of a single vocabularly, and not introducing new terms for concepts that already have a name, can this field be named `liveUntilLedgerSeq`?